### PR TITLE
more performant nav bar

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

This is an intentional change to allow the nav bar to act properly when the screen is resized.

Before, the dropdown would appear and eventually go away which was annoying

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3997** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3997" target="_blank">Graphite</a>  👈
    * **PR #3998** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3998" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3997?utm_source=stack-comment).

File: src/components/Navbar/SiteNavigation.tsx [hunk 1]
```diff
@@ -13,88 +13,160 @@ import { TransactionsList } from './components/TransactionList'
...
+              <WalletButton />
+            </div>
+          </div>
+        </>
+      </nav>
+    </div>
+  )
+}
+
+const MobileSitNavigation = () => {
```
Comment (peripheralist) [inline src/components/Navbar/SiteNavigation.tsx:93]: typo lulz

**Review by peripheralist — APPROVED** [review]

**Comment by wraeth-eth** [comment]
@wraeth-eth started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3997).

**Comment by wraeth-eth** [comment]
[Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3997) rebased this pull request as part of a merge.

**Comment by wraeth-eth** [comment]
@wraeth-eth merged this pull request with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3997).

**Pull request merged**